deploy/observe-server-gn-test/src/main/resources/app/environments.conf.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "hostName": "observe.hi.gemini.edu",
+    "hostName": "*",
```

Comment:
Can you explain this change?

Reply:
This is legacy logic from explore, where we use the host as a discriminator to know what environment we are in. Here we just have a different file for each environment. 

In any case, these files are going away soon since we plan to pass these parameters from the server to the client.